src/SpectatorView.Unity/Assets/SpatialAlignment.ASA/Scripts/SpatialAlignment/SpatialAnchorsCoordinateService.cs

```diff
@@ -120,6 +130,11 @@ public override async Task<bool> TryDeleteCoordinateAsync(string id, Cancellatio
             }
         }
 
+        private async Task<bool> TryDeleteCoordinateEditorAsync(string id, CancellationToken cancellationToken)
+        {
+            return await Task.FromResult(true);
```

Comment:
>return await Task.FromResult(true); [](start = 12, length = 35)

You could remove the "async" from this method and just return "Task.FromResult(true)" without the await